lib/pbench/server/api/resources/query_apis/metadata_index/namespace_and_rows.py

```diff
@@ -233,13 +225,14 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
                 "run_id": run id of a dataset that client wants to search for
                         fetching the iteration samples.
 
-                "scroll_id": Server provided Elasticsearch scroll id that client
-                            recieved in the result of the original query with
-                            filters. This can be used to fetch the next page of
-                            the result.
+                "scroll_id": Optional Server-provided Elasticsearch scroll id
```

Comment:
I don't think `Server-provided` means anything here; or at least the context is so convoluted that it doesn't help. Simply saying "Elasticsearch scroll id" would be better.